subscription-manager.spec

```diff
@@ -63,6 +63,7 @@
 %global rhsm_package_name %{py_package_prefix}-subscription-manager-rhsm
 %define include_syspurpose 1
 %else
+%global python_sitearch %python2_sitearch
```

Comment:
You cut set __python to %__python2 instead to change all similar macros as well. At least in Fedora.